.github/workflows/csharp.yml

```diff
@@ -38,7 +37,7 @@ jobs:
       TRAVIS_COM_TOKEN: ${{ secrets.TRAVIS_COM_TOKEN }}
   
   fullstack_production_suite:
-    name: Run Full Stack Compatibility Suite
+    name: Run Optimizely Feature Experimentation Compatibility Suite
```

Comment:
@mnoman09 I wonder if @russell-loube-optimizely is on point here.  The ping is not completing b/c we've renamed the job.  Maybe revert this particular instance.

Reply:
No this does not have to do with the issue because build uses fullstack_production_suite as ID and it is running fine.

Reply:
Cool.  I'll merge then.